samples/test/placeholder_if.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2020 The Kubeflow Authors
+# Copyright 2020,2021 The Kubeflow Authors
```

Comment:
nit: usually we don't update or add this.

Reply:
I don't have a strong opinion, looks like it's optional